### ISSUE TITLE
Difference size of `rgb` and `origins` lead to `Index out of` error

### ISSUE THREAD

**Issue by FavorMylikes** [google/nerfies#28]
![image](https://user-images.githubusercontent.com/16064851/129345735-8dd505b5-2862-49f4-ad27-1d62f4348c66.png)
![image](https://user-images.githubusercontent.com/16064851/129345822-4ed7762d-8ce7-4202-ad71-c5650200ebdb.png)

`shuffled_inds` between `0 to `120*68*11=89760`
but x has range only from `0` to `71*40*3*11=31240`

I don't know how to reduce the size of `rgb` data or the other

**Comment by keunhong** [comment]
It seems like your cameras do not match your images for some reason. They are quite small so it might be a rounding issue.

**Comment by SangbumChoi** [comment]
@FavorMylikes @keunhong I've found that this error comes when I put train_dataset as video of Mac(no error at this point) or iPhone(error). Maybe resize issue?
